### PR TITLE
Display completed (sent and unsent) letters on My Letters page

### PR DESCRIPTION
**Pull request by shakao**
For [sc-9613]

- Adds a list of all created letters to the user session
- Renders this list on the My Letters Page
- Update tests, strings

Screen Shot 2022-06-24 at 11 28 47 AM Screen Shot 2022-06-24 at 11 27 01 AM

TODO:
- Download button currently always downloads the PDF for the latest letter--I think it might be difficult with our current infrastructure to change this, will look into it but we may also consider falling back to just displaying a list of issues from that letter
- List of access dates not currently hooked up
- Needs final CSS pass to match mocks

### CODE REVIEW

**Comment by shakao** [comment]
will wait til https://github.com/JustFixNYC/tenants2/pull/2319 is merged to merge this one, since there will probably be a couple `messages.po` conflicts

File: frontend/lib/laletterbuilder/letter-builder/my-letters.tsx [hunk 1]
```diff
@@ -22,12 +26,140 @@ export const LaLetterBuilderMyLetters: React.FC<ProgressStepProps> = (
...
+        });
+        return (
+          <CompletedLetterCard
+            key={`sent-letter-${i}`}
+            link={
+              LaLetterBuilderRouteInfo.getLocale("en").habitability
+                .letterContent.pdf
+            }
+          >
+            <h3>{`JustFix sent your letter on ${dateString} `}</h3>
```
Comment (austensen) [inline frontend/lib/laletterbuilder/letter-builder/my-letters.tsx:142]: Does this need `li18n._(t``)` ?

**Review by austensen — APPROVED** [review]
This all looks great! Just noticed one place that might need a translation

File: frontend/lib/laletterbuilder/letter-builder/my-letters.tsx (hunk 1 above)
Comment (sraby) [inline frontend/lib/laletterbuilder/letter-builder/my-letters.tsx:142]: Does this h3 text need to be internationalized?

**Pull request merged**